### PR TITLE
[*] CORE : Improve configuration load & get performances

### PR DESCRIPTION
**Pull request by jocel1**

### CODE REVIEW

**Comment by xBorderie** [comment]
Thank you @jocel1 !
Any idea how we could be test this? :)

File: classes/db/Db.php [hunk 1]
```diff
@@ -602,6 +602,7 @@ public function executeS($sql, $array = true, $use_cache = true)
             }
         }
 
+
```
Comment (mickaelandrieu) [inline classes/db/Db.php]: can you remove this line please ?

File: tests/Integration/classes/ConfigurationCoreTest.php [hunk 2]
```diff
@@ -24,7 +24,7 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
-namespace PrestaShop\PrestaShop\Tests\Integration\Classes;
+namespace PrestaShop\PrestaShop\tests\Integration\classes;
```
Comment (mickaelandrieu) [inline tests/Integration/classes/ConfigurationCoreTest.php]: nice catch

**Comment by jocel1** [comment]
@xBorderie : if the tests all passed, it should be ok (it's basically used in every tests :))

**Comment by mickaelandrieu** [comment]
@xBorderie let's wait for @jocel1, I need to discuss with him some details :)

**Pull request merged**